.github/workflows/cd-production.yml

```diff
@@ -3,12 +3,13 @@ on:
   push:
     branches:
       - beta-release
+      - release
```

Comment:
I asked dev leads to add branch protection for the `release` branch (most importantly, require PRs). Along with `enforce-release-workflow` in `ci-policies.yml`, that should ensure only PRs from `master` can be pushed to the `release` branch. Then, we can make the switch and stop using the `beta-release` branch for prod deployments 😄

Reply:
WOOHOO!

Reply:
Branch protection is now extended to all branches with suffix `release`